spec/models/embed/was_seed_thumbs_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+describe Embed::WasSeedThumbs do
+  include WasSeedThumbsFixtures
+  before :each do
+  end
+  describe 'initialize' do
+    it 'initializes the WasSeedThumbs with the druid' do
+      seed_thumbs = Embed::WasSeedThumbs.new('ab123cd4567')
+      expect(seed_thumbs.druid).to eq('ab123cd4567')
+    end
+  end
+  
+  describe 'get_thumbs_list' do
+    it 'returns a list of thumbs' do
+      allow_any_instance_of(Embed::WasSeedThumbs).to receive(:response).and_return(thumbs_list)
+
+      seed_thumbs = Embed::WasSeedThumbs.new('ab123cd4567')
+      thumbs_list = seed_thumbs.get_thumbs_list
+      expect(thumbs_list.length).to eq(2)      
+      expect(thumbs_list[0]).to eq({
+        "memento_uri"=> "https://swap.stanford.edu/20121129060351/http://naca.central.cranfield.ac.uk/",
+        "memento_datetime"=> "20121129060351",
+        "thumbnail_uri"=> "https://stacks.stanford.edu/image/iiif/gb089bd2251%2F20121129060351/full/200,/0/default.jpg"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.